src/term/transform/encode_pattern_matching.rs

```diff
@@ -286,7 +302,11 @@ fn get_pat_arg_count(match_path: &[Pattern]) -> (usize, usize) {
     Pattern::Var(_) => 1,
     Pattern::Ctr(_, vars) => vars.len(),
     Pattern::Num(_) => todo!(),
-    Pattern::Tup(..) => todo!(),
+    Pattern::Tup(fst, snd) => match (fst, snd) {
+      (None, None) => 0,
```

Comment:
With the way the encoding is implemented right now, we always pass the unused arguments forward, so tuples should always count as 2 variables. We can improve this later to insert some Era terms at the end, but for now fix this block

Reply:
Also, we should have some tests for compiling unused pattern variables